docs/libtorch.rst

```diff
@@ -34,3 +34,24 @@ To produce libtorch.a rather than libtorch.so, set the environment variable `BUI
 To use ninja rather than make, set `CMAKE_GENERATOR="-GNinja" CMAKE_INSTALL="ninja install"`.
 
 Note that we are working on eliminating tools/build_pytorch_libs.sh in favor of a unified cmake build.
+
+Building libtorch using CMake
+--------------------------------------
+
+You can build C++ libtorch.so directly with cmake.  For example, to build a v1.6.0 Release version that uses CUDA and CUDDN, and install it in the directory specified by CMAKE_INSTALL_PREFIX below, you can use
+::
+   git clone https://github.com/pytorch/pytorch.git
+   cd pytorch
+   git checkout v1.6.0
+   git clean -fdx
+   git submodule update --init --recursive
+   mkdir ../pytorch-build
+   cd ../pytorch-build
+   cmake -DBUILD_SHARED_LIBS:BOOL=ON -DCMAKE_BUILD_TYPE:STRING=Release -DPYTHON_EXECUTABLE=`which python3` -DUSE_CUDA:BOOL=ON -DUSE_CUDNN:BOOL=ON -DCMAKE_INSTALL_PREFIX:PATH=../pytorch-install ../pytorch
```

Comment:
Is there a reason why you specify variable types in some cases (i.e. `-DBUILD_SHARED_LIBS:BOOL=ON`), but do not in others (i.e. -DPYTHON_EXECUTABLE=foo vs -DPYTHON_EXECUTABLE:PATH=FOO)

Reply:
I will change that to `-DPYTHON_EXECUTABLE:PATH=FOO`.